pkg/handlers/generic/lifecycle/csi/aws-ebs/handler.go

```diff
@@ -114,13 +117,17 @@ func (a *AWSEBS) createStorageClasses(ctx context.Context,
 		}
 		params := defaultParams
 		if c.Parameters != nil {
-			params = c.DeepCopy().Parameters
+			m := make(map[string]string)
+			for k, v := range c.Parameters {
+				m[k] = v
+			}
+			params = m
```

Comment:
`maps.Clone` ftw https://pkg.go.dev/maps@go1.22.1#Clone

```suggestion
			params = maps.Clone(c.Paramaters)
```

Reply:
nice!